cli/azd/pkg/project/framework_service_noop.go

```diff
@@ -0,0 +1,69 @@
+package project
```

Comment:
This exists to work around the fact that the Docker framework service wants an inner framework service (so `azd restore` and `azd build` could do `npm restore` or `npm build` for you).

In aspire, we have no idea about what the underlying language is - so we just need a thing to opt out of this.

I did consider changing framework_service_docker to handle the case where the inner framework service was nil (or just set this as the inner framework service by default in `NewDockerProject`) and have the importer set the language to Docker, but that felt worse and still seemed like it would still require some surgery in  `(*serviceManager).GetFrameworkService` so I landed here.

Feels like we might be able to clean this up after some refactorings, but I wanted to keep this diff somewhat small and this seemed like the best way to accomplish that.

@wbreza FYI, I think I recall seeing some drafts from you at some point with a `none` language. Wondering if this would be helpful to you once I land it.

Reply:
Cleaned some of this stuff up, I like how this is structured now.  This project still exists because I think it will be useful, but I could imagine just removing this in favor of making the dockerProject careful about where it assumes framework would be non nil.